### ISSUE TITLE
[Question] Redundant methods

### ISSUE THREAD

**Issue by longnd** [aungkoko1234/web-scrapper-backend#18]
Issue
---
The Helper has several unused methods, for example,
https://github.com/aungkoko1234/web-scrapper-backend/blob/9c520dc93598646fcc0f72d54845ec7a2838a985/src/helpers/helpers.service.ts#L39

https://github.com/aungkoko1234/web-scrapper-backend/blob/9c520dc93598646fcc0f72d54845ec7a2838a985/src/helpers/helpers.service.ts#L72

they're irrelevant to the context of this small application. Are they being used somewhere?

**Comment by aungkoko1234** [comment]
Yeah they are not necessary. I forgot to remove those after I tested. I will remove those.

**Comment by longnd** [comment]
may I know why the application needs those functions and why did you need to test them?

**Comment by aungkoko1234** [comment]
For image delete function, I intended to delete the csv file after the queue for searching keyword is complete. Even though the naming is wrong.

**Comment by aungkoko1234** [comment]
@longnd  I have remove those redundant methods.
